### PR TITLE
add french support

### PR DESCRIPTION
**Pull request by LioraLevy**
I added support for french in addition to other languages provided my the ministry of health

As an active member of the community of Olim for France in Israel , I am constantly updating french Olim who don't speak hebew and hence do not have access to Ministry of Health instructions ( which are provided in 5 languages but not in french)

This is a very small modification for the app that can help many new immigrants to have full access to ministry of Health instructions and recommandations and contribute to saving many lives in our country.

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -87,21 +93,27 @@ const config: AppIds = {
...
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en/',
     },
     reportForm: {
       he: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       en: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       am: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       ru: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
-      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il'
+      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en/',
```
Comment (lidorbt) [inline src/config/config.ts]: That's not the right link

File: src/config/config.ts [hunk 2]
```diff
@@ -137,21 +149,27 @@ const config: AppIds = {
...
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en',
     },
     reportForm: {
       he: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       en: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       am: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
       ru: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
-      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il'
+      ar: 'https://govforms.gov.il/mw/forms/QuarantineForExposees%40health.gov.il',
+      fr: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-en',
```
Comment (lidorbt) [inline src/config/config.ts]: same here, that's not the right link

File: src/config/config.ts (hunk 1 above)
Comment (LioraLevy) [reply src/config/config.ts]: Bc there is no link in french on the ministry website !

Comment (LioraLevy) [reply src/config/config.ts]: So english is better than nothing

File: src/config/config.ts (hunk 2 above)
Comment (LioraLevy) [reply src/config/config.ts]: Same. 
No french on the ministry website so english is better than nothing

**Comment by MiriShalev** [comment]
Thanks @LioraLevy !  The MoH team would have to advise before adding French.

**Comment by vcorem** [comment]
Will you publish compiled APK for non-Google Play phones?
(e.g https://grapheneos.org/)

File: src/locale/texts.json [hunk 3]
```diff
@@ -498,5 +508,107 @@
...
+      "subTitle2": "Un diagnostic et un isolement en temps opportun peuvent leur sauver la vie"
+    },
+    "location": {
+      "title": "La localisation est necessaire",
+      "subTitle1": "Cette application va collecter vos données de localisation et les comparer aux emplacements des personnes connues comme etant infectées par le virus. Si l'un de ces emplacements se chevauche, vous recevrez une notification.",
+      "subTitle2IOS": "Pour que cela fonctionne, vous devez activer le service de localisation sur votre téléphone.",
+      "subTitle2Android": "Pour que cela fonctionne, vous devez avoir à tout moment des services de localisation activés sur votre téléphone.",
+      "dataAnonymous": "\nVos données de localisation seront rendues anonymes",
+      "consent1": "J'ai lu et j'approuve les ",
+      "consent2": "conditions d'utilisation",
```
Comment (ybaruchel) [inline src/locale/texts.json]:
```suggestion
      "consent2": "Conditions d'utilisation",
```

**Comment by MiriShalev** [comment]
Hey @LioraLevy  and @ybaruchel, I'm happy to share that the MoH team decided to add French and to accept your version! We'll add it to the next release.

**Pull request closed without merging**